justfile

```diff
@@ -286,8 +286,23 @@ download-assets:
 check-android-so:
     {{justfile_directory()}}/.github/scripts/check_android_so.sh "$FLUTTER_WORKSPACE"/android/src/main/jniLibs/
 
+_override-flutter-self-deps $BUILD_ID:
+    #!/usr/bin/env bash
+    set -eux
+    cd "$FLUTTER_EXAMPLE_WORKSPACE"
+
+    SED_CMD="sed"
+    if [[ "{{os()}}" == "macos" ]]; then
+        SED_CMD="gsed"
+    fi
+
+    # This will add changes to your repo which should never be committed.
+    $SED_CMD -i s/dependency_overrides/x_dependency_overrides/ ./pubspec.yaml
```

Comment:
❓ does the `x_` just modifies the name such that flutter won't recognize this item as the `dependency_overrides`? if so, should we give it a more readable name, maybe sth lik `DISABLED_dependency_overrides`?

Reply:
yes it just makes the entry unrecognizable for dart, it's the same as we do in the release script.

Reply:
this is also not disabling dependency overrides per-se, just making them unrecognizable for dart.

In the end it's an ugly hack, but doing it properly isn't worth the time needed to make it work.

This is not something you should ever commit to the repo, it's just temporary for debugging purpose. Hence why I also made the job start with a `_`.